### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by iYared**
Changed CalGet emoji from Star to Calendar, since it's relate with the service

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -242,12 +242,13 @@ Total deals: 110
 |----|-----------| ------ |---------------------------------------------------------------------------------------------------|
 |  ⭐ | [Uneed](https://uneed.best) | Product Hunt alternative. | 30% off "Skip the waiting line" with code **BLACKFRIDAY2024** to promote your product.
 |  ⭐ | [Redreach](https://redreach.ai) | Reddit Marketing: Turn Reddit traffic into customers. Finds high impact Reddit conversations to help you naturally promote your product and get more sales. | 30% OFF with code **BLACKFRIDAY24** (both 3 Day Access Plan and all recurring plans)
+|  📆 | [CalGet](https://calget.com) | Simplify your event planning with **CalGet**. Create shareable “Add to Calendar” links that integrate seamlessly into your social media, newsletters, websites, and emails. With just one click, your audience can add events to their preferred calendars, including Google, Apple, Outlook, Yahoo, and Office365. Streamline your event promotion and boost attendance with CalGet.com.| 30% OFF on All YEARLY plans with code **BLACKFRIDAY2024** |
```
Comment (dqhieu) [inline README.md:245]: Hi @iYared , please don't change the position of the deal. Thanks

Comment (iYared) [reply README.md:245]: Hi @dqhieu We were 3rd position on our first pull request, but encountered an issue with it.

Comment (iYared) [reply README.md]: Hi @dqhieu , I've resolved the conflict

**Comment by gh-readonly-tdinh-me** [comment]
too much unrelated diff, please check again

**Pull request closed without merging**